src/bundle/Resources/views/content/tabs/locations.html.twig

```diff
@@ -4,6 +4,7 @@
 <div class="ez-list-toolbar">
     <h2 class="ez-list-toolbar-label">{{ 'locationview.locations.content.locations'|trans()|desc('Content locations') }}</h2>
     <div class="container-buttons">
+        {{ form_widget(actionsForm.add, {'label': 'locationview.locations.add.location'|trans()|desc('Add Location'), 'attr': {'class': 'ez-button ez-button-secondary'}}) }}
```

Comment:
so following my previous comment, you should also render `actionsForm.parentLocationId` here